include/nlohmann/detail/meta/type_traits.hpp

```diff
@@ -54,11 +54,6 @@ struct is_basic_json_context :
     || std::is_same<BasicJsonContext, std::nullptr_t>::value >
 {};
 
-template<typename> struct is_json_pointer : std::false_type {};
```

Comment:
Is this no longer needed?

Reply:
It's still needed but I had a different implementation in this branch before I added this one with the `json_pointer` refactor. The implementation in this branch introduces a generic `is_specialization_of` trait. Using this trait `is_json_pointer` becomes a simple alias template declaration and other traits can later be rewritten/shortened as well.

Reply:
Thanks for clarifying.